terraform/civil.tf

```
@@ -0,0 +1,19 @@
+locals {
+  cv_non_prod_domains = ["mail-civil-nonprod.platform.hmcts.net"]
+  cv_prod_domains     = ["mail-civil-prod.platform.hmcts.net"]
```

cv isn't a known or useful abbreviation, I would suggest expanding it: (will need other changes)
```suggestion
  civil_prod_domains     = ["mail-civil-prod.platform.hmcts.net"]
```

---

done